<?xml version="1.0"?>

<launch>
  <arg name="name" default="OHRAD_X3"/>
  <group ns="$(arg name)">

    <node name="joy_teleop_mixer" type="joy_teleop_mixer" pkg="nearness_control" output="screen">
      <remap from="joy_teleop_mixer/joy" to="joy"/>
      <remap from="joy_teleop_mixer/cmd_vel" to="cmd_vel"/>

      <param name="max_forward_speed" value="1.5"/>
      <param name="max_lateral_speed" value="1.5"/>
      <param name="max_vertical_speed" value=".25"/>
      <param name="max_yaw_rate" value=".5"/>

    </node>

  </group>

</launch>
